openedx/features/course_duration_limits/migrations/0008_auto_20190313_1634.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-03-13 16:34
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import openedx.core.djangoapps.config_model_utils.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('course_duration_limits', '0007_auto_20190311_1919'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='coursedurationlimitconfig',
+            name='org_course',
+            field=models.CharField(blank=True, db_index=True, help_text="Configure values for all course runs associated with this course. This is should be formatted as 'org+course' (i.e. MITx+6.002x, HarvardX+CS50).", max_length=255, null=True, validators=[openedx.core.djangoapps.config_model_utils.models.validate_course_in_org], verbose_name='Course in Org'),
```

Comment:
my understanding is that MITx+6.002x is the course id and that although convention dictates it should be org+course that it is not always the case and there are course run keys where the course is replaced by something else.  
i know the specific examples i recall were quite old, so perhaps this is now the case for all runs, but I wasn't sure if instead maybe the instructions should say to include the discovery course key for the course run

Reply:
As it stands now, this code isn't actually talking to discovery, it's just manipulating the course-run-key to get data in the right format. So, if there are courses where the course-key differs from the data in the course-run-key, this code won't get it right, at the moment.

Reply:
actually i was thinking of course runs key, so this code may be fine
Take this as an example
https://prod-edx-discovery.edx.org/api/v1/course_runs/ANUx/ANU-ASTRO1x/1T2014/

course_run_key: ANUx/ANU-ASTRO1x/1T2014/
course_key: ANUx+ANU-ASTRO1x

I don't know what ANU-ASTRO1x corresponds to but is that value what your code needs to work?

Reply:
IN the context of that key (an old-style course run key), the correct format would be "ANUx+ANU-ASTRO1x".